src/pages/MapPage.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  IonButton,
+  IonButtons,
+  IonContent,
+  IonHeader,
+  IonIcon,
+  IonMenuButton,
+  IonModal,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+} from "@ionic/react";
+import { listSharp } from "ionicons/icons";
+import { useRef } from "react";
+import InventoryList from "../components/InventoryList";
+
+import MainMap from "../components/MainMapMaplibre";
+import { useSettings } from "../context/settings";
+
+const MapPage: React.FC = () => {
+  // modal reference
+  const modalRef = useRef<HTMLIonModalElement>(null);
+
+  return (
+    <IonPage>
+      <IonHeader>
+        <IonToolbar>
+          <IonButtons slot="start">
+            <IonMenuButton />
+          </IonButtons>
+          <IonTitle>Mathislewand</IonTitle>
+        </IonToolbar>
+      </IonHeader>
+
+      <IonContent fullscreen>
+        <MainMap />
+
+        {/* <IonModal
+          ref={modalRef}
+          breakpoints={[0, 1.0]}
+          initialBreakpoint={1.0}
+          trigger="list-modal"
+          backdropBreakpoint={1.0}
+          style={{
+            maxWidth: "600px",
+            left: "calc(100% - 600px)",
+            position: "absolute",
+            marginTop: "46px",
+          }}
+        >
+          <IonHeader>
+            <IonToolbar>
+              <IonTitle>Inventory List</IonTitle>
+              <IonButtons
+                slot="end"
+                onClick={() => modalRef.current?.dismiss()}
+              >
+                <IonButton>Close</IonButton>
+              </IonButtons>
+            </IonToolbar>
+          </IonHeader>
+          <IonContent>
+            <InventoryList />
+          </IonContent>
+        </IonModal> */}
```

Comment:
Let's get rid of this. With this PR merged, there is no way back to the modal layout, so we don't need to keep this code.